cmd/ncproxy/hcn.go

```diff
@@ -206,25 +234,34 @@ func createHCNNetwork(ctx context.Context, req *ncproxygrpc.HostComputeNetworkSe
 
 func hcnNetworkToNetworkResponse(network *hcn.HostComputeNetwork) (*ncproxygrpc.GetNetworkResponse, error) {
 	var (
-		ipamType                int32
-		defaultGateway          string
-		switchName              string
-		subnetIPAddressPrefixes []string
+		ipamType                  int32
+		defaultGateway            string
+		defaultGatewayIPv6        string
+		switchName                string
+		subnetIPAddressPrefixes   []string
+		subnetIPv6AddressPrefixes []string
 	)
 
 	for _, ipam := range network.Ipams {
+		// all ipams should havee the same type
+		ipamType = ncproxygrpc.HostComputeNetworkSettings_IpamType_value[ipam.Type]
```

Comment:
is this repeatedly setting `ipamType` for each `ipam`, and just keeping the last value?

Reply:
Yes, but it's expected that all the ipams should be the same. I could move this out if you'd prefer.

Reply:
its fine as is, but can you add a comment blurb saying were just keeping the last value?
i am curious why none of the linters didnt complain about ineffective assignment ...